Server/src/routes/tracker.route.ts

```diff
@@ -172,7 +172,7 @@ export class TrackerRoute {
 			res.sendStatus(200)
 			return
 		}
-		const timestamp = new Date(trackerData.received_at)
+		const timestamp = new Date()
```

Comment:
This maybe seems not that important and also a bit counter intuitive as this _should_ be the same. However while testing I ran into problems, that the trackers data said it got received two hours earlier than my system time, which gets used in the Date-class. So to make this consistent and within one time-system I changed that to just use the Date-class default as well (also we are mostly just interested in intervals and not receiving times).

_One further note: I just checked, that the vehicle simulator, which is used for testing, uses UTC time and as stated above TypeScripts Date-class uses system time. So that is exactly the problem, but instead of changing the vehicle simulator I still suggest the change here for other trackers in the future (maybe even for app users from other countries)._